src/libponyc/codegen/genexe.c

```diff
@@ -277,6 +278,9 @@ static bool link_exe(compile_t* c, ast_t* program,
 #ifdef PONY_USE_LTO
     "-flto -fuse-linker-plugin "
 #endif
+#if defined(PLATFORM_IS_ALPINE_LINUX) || defined(PLATFORM_IS_FREEBSD)
```

Comment:
Quick thought: could `PLATFORM_IS_ALPINE_LINUX` be named  `PLATFORM_IS_MUSL_LINUX`, and still be correct?